### ISSUE TITLE
Unable to generate Prisma Client with aggregateApi enabled

### ISSUE THREAD

**Issue by MichalLytek** [prisma/prisma-client-js#772]
## Bug description

When I add `experimentalFeatures = ["aggregateApi"]` to my Prisma schema file, it stops generating properly and instead throws me this error:

```
Error: 
TypeError: Cannot read property 'name' of undefined
    at new type (/home/majkel/development/typegraphql-prisma/experiments/node_modules/@prisma/client/dist/generation/TSClient.js:1057:1)
    at aggregateTypes (/home/majkel/development/typegraphql-prisma/experiments/node_modules/@prisma/client/dist/generation/TSClient.js:646:1)
    at Array.map (<anonymous>)
    at Model.getAggregationTypes (/home/majkel/development/typegraphql-prisma/experiments/node_modules/@prisma/client/dist/generation/TSClient.js:645:1)
    at Model.includes [as toTS] (/home/majkel/development/typegraphql-prisma/experiments/node_modules/@prisma/client/dist/generation/TSClient.js:730:3)
    at dmmf (/home/majkel/development/typegraphql-prisma/experiments/node_modules/@prisma/client/dist/generation/TSClient.js:282:1)
    at Array.map (<anonymous>)
    at TSClient.values [as toTS] (/home/majkel/development/typegraphql-prisma/experiments/node_modules/@prisma/client/dist/generation/TSClient.js:281:3)
    at Object.JS (/home/majkel/development/typegraphql-prisma/experiments/node_modules/@prisma/client/dist/generation/TSClient.js:24:8)
    at fileMap (/home/majkel/development/typegraphql-prisma/experiments/node_modules/@prisma/client/dist/generation/generateClient.js:48:1)
```

## How to reproduce

```sh
npx prisma generate
```

## Expected behavior

Generate the client that supports new `aggregate` operation.

## Prisma information

```prisma
datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

type Numeric = Float

generator client {
  provider             = "prisma-client-js"
  binaryTargets        = ["windows", "debian-openssl-1.1.x"]
  output               = "../prisma/generated/client"
  experimentalFeatures = ["aggregateApi"]
}

model post {
  uuid      String  @default(uuid()) @id
  published Boolean
  title     String
}
```

## Environment & setup

- OS: Ubuntu 20.04 (WSL2) 
- Database: PostgreSQL 
- Prisma version: 2.2.0 
- Node.js version: 13.10

**Comment by MichalLytek** [comment]
@timsuchanek It's happening when the model name is lowercase and that's happening for all users who introspect their postgres db 😉

**Comment by Jolg42** [comment]
Thanks a lot for reporting 🙏

This issue is fixed in the latest version of @prisma/cli: 2.2.1
